src/main/resources/META-INF/hodgepodge_at.cfg

```diff
@@ -19,3 +19,4 @@ public net.minecraft.client.resources.AbstractResourcePack field_110597_b # reso
 public net.minecraft.client.resources.FallbackResourceManager field_110540_a # resourcePacks
 public net.minecraft.client.resources.FileResourcePack field_110600_d # resourcePackZipFile
 public net.minecraft.client.resources.SimpleReloadableResourceManager field_110548_a # domainResourceManagers
+public net.minecraft.world.WorldServer func_73053_d()V # wakeAllPlayers()
```

Comment:
Why use an AT aren't we team mixin accessors

Reply:
No we aren't.

Reply:
ok :(